src/utils/configValidators/createUnique.js

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export default () => {
+  const values = [];
```

Comment:
Even better!

Reply:
Will this be tied to a specific key?

Reply:
Yes, each key has its own unique set.  So if you had a property id that matched a ims org id, it would still validate.  When you call `createUnique()`, it creates a new values array.  So when this is called as part of the data collector validator initialization it will create one instance for propertyId, and one for imsOrgId.